provider/pkg/gen/overlays.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2020, Pulumi Corporation.
+// Copyright 2016-2021, Pulumi Corporation.
```

Comment:
You can ignore most of the changes in this file. I refactored the declaration into separate variables, but it's not a functional change.